README.md

```diff
@@ -1,35 +1,62 @@
 
-```bash
-$ juju deploy zookeeper -n 3
-$ juju deploy kafka -n 3
-$ juju relate kafka zookeeper
-```
-
-## A fast and fault-tolerant, real-time event streaming platform!
-Manual, Day 2 operations for deploying and operating Apache Kafka, topic creation, client authentication, ACL management and more are all handled automatically using the [Juju Operator Lifecycle Manager](https://juju.is/docs/olm).
-
-### Key Features
+This operator charm comes with features such as:
+- Fault-tolerance, replication, scalability and high-availability out-of-the-box.
 - SASL/SCRAM auth for Broker-Broker and Client-Broker authenticaion enabled by default.
 - Access control management supported with user-provided ACL lists.
-- Fault-tolerance, replication and high-availability out-of-the-box.
-- Smooth topic-creation through [Juju Actions](https://juju.is/docs/olm/working-with-actions) and [application relations](https://juju.is/docs/olm/relations)
 
-### Secure-by-default, authenticated cluster administration
-Apache Kafka ships with `bin/*.sh` commands to do various administrative tasks, e.g `bin/kafka-config.sh` to update cluster configuration, `bin/kafka-topics.sh` for topic management, and many more! The Kafka Charmed Operator provides these commands to administrators to easily run their desired cluster configurations securely with SASL authentication, either from within the cluster or as an external client.
+The Kafka Operator uses the latest upstream Kafka binaries released by the The Apache Software Foundation that comes with Kafka, made available using the [`charmed-kafka` snap ](https://snapcraft.io/charmed-kafka) distributed by Canonical.
+
+As currently Kafka requires a paired ZooKeeper deployment in production, this operator makes use of the [ZooKeeper Operator](https://github.com/canonical/zookeeper-operator) for various essential functions.
+
+### Checklist
+- [x] Super-user creation
+- [x] Inter-broker auth
+- [x] Horizontally scale brokers
+- [x] Username/Password creation for related applications
+- [x] Automatic topic creation with associated user ACLs
+- [x] Persistent storage support with [Juju Storage](https://juju.is/docs/olm/defining-and-using-persistent-storage)
+- [x] TLS/SSL encrypted connections
+- [ ]  mTLS
+- [ ] Multi-application clusters
+- [ ] Partition rebalancing during broker scaling
+
+## Requirements
+
+For production environments, it is recommended to deploy at least 5 nodes for Zookeeper and 3 for Kafka.
+While the following requirements are meant to be for production, the charm can be deployed in much smaller environments.
+
+- 64GB of RAM
+- 24 cores
+- 12 storage devices
+- 10 GbE card
 
-To run commands from within the cluster, the charm makes available a `client.properties` file with all the necessary configuration to run a `KafkaClient`. One can retrieve `admin` credentials with:
+## Usage
```

Comment:
would be nice to have a simple config options section (this includes configurations set by actions) feel free to blatantly [copy these](https://github.com/canonical/mongodb-operator/blob/1d1d90610264e9f50e72c92b37018464847b56f8/README.md?plain=1#L19-L27)

Reply:
This is an interesting point. We are planning to have quite a lot of config options. Should all of these be shown here? Charmhub already picks up config options, with their descriptions, on a specific tab, same with actions.

Reply:
yes, I agree with @zmraul . I understand that Nick mentioned this in his email, but I don't think it is scalable to do this (especially here that we will have 30 config options) and I believe we have a way forward (having these referenced to charmhub)

I believe in normal charms this may still be all right (although a bit redundant), but not for us.  Let's not do this!